CHANGELOG.md

```diff
@@ -1,3 +1,155 @@
+# v1.13.0 (2022-03-15)
+
+## OS Changes
+
+* Set NVMe IO request timeouts for EBS according to AWS recommendations ([#2820])
+* Add `ethtool` to Bottlerocket ([#2829])
+* rpm2img: stripe root filesystem for measureable performance improvement during boot ([#2863]) ([#2879])
+* Label `BOTTLEROCKET-DATA` at runtime ([#2807])
+* Add info logging to migrator in order to document onging migrations ([#2751])
+* Add `CAP_SYS_MODULE` and `CAP_CHROOT` to bootstrap containers ([#2772])
+* Add support for cgroup v2 ([#2875]) ([#2802])
+* Disable IA and SafeSetID LSM for kernel-5.15 ([#2789])
+* Update kernel-5.10 to 5.10.165 and kernel-5.15 to 5.15.90 ([#2795])
+* Allow `=` in bootconfig values ([#2806])
+* Ensure 'prepare-boot' runs after 'mark-successful-boot' ([#2873])
```

Comment:
This PR should be listed in the same line item as "Label `BOTTLEROCKET-DATA` at runtime ([#2807])"